go.mod

```diff
@@ -20,54 +20,50 @@ require (
 	github.com/hashicorp/vault/api v1.0.5-0.20191108163347-bdd38fca2cff
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/mitchellh/hashstructure v1.0.0
-	github.com/onsi/ginkgo/v2 v2.5.0
+	github.com/onsi/ginkgo/v2 v2.6.0
 	github.com/onsi/gomega v1.24.1
 	github.com/pkg/errors v0.9.1
-	github.com/pseudomuto/protoc-gen-doc v1.5.1
+	github.com/pseudomuto/protoc-gen-doc v1.5.0
```

Comment:
do you know why this got downgraded?

Reply:
Hmm not sure,  i'll check go-utils & k8s-utils to see which might've done it :hmm:

Reply:
Yeah not sure. looks like it was part of [this commit](https://github.com/solo-io/solo-kit/pull/542/commits/eb2501fe78e49efa241aaa3ae9871c5536a52d8e) where i bumps k8s-utils to a specific commit before release. I think I can `go get github.com/pseudomuto/protoc-gen-doc@v1.5.1` if you think that'd be safer